client/app/license/order/license-order.html

```diff
@@ -237,12 +237,13 @@ <h2 data-translate="license_order_step4_title"></h2>
                         </div>
                     </div>
 
-                    <div class="col-md-3"
+                    <div class="m-4"
```

Comment:
Couldn't be moved outside of the `.row` then?

Because you are not respecting this following rules: https://github.com/twbs/bootlint/wiki/E013.

Hope its helps you!

Reply:
Cant move it out of row so added `col-md-12`, so that it appears in next line